src/object/cli_obj.c

```diff
@@ -24,7 +24,9 @@
 #define CLI_OBJ_IO_PARMS	8
 #define NIL_BITMAP		(NULL)
 
-#define OBJ_TGT_INLINE_NR	(11)
+#define OBJ_TGT_INLINE_NR	11
```

Comment:
When you resolved the merge conflict, this changed from 12 to 11.   Is that intended?

Reply:
The value for OBJ_TGT_INLINE_NR on the latest master is (11). My patch needs not to change such value. It just removes the redundant parenthesis.